docs/utilities/validation.md

```diff
@@ -199,3 +199,31 @@ You can use our built-in [JMESPath functions](./jmespath_functions.md){target="_
 
 ???+ info
     We use these for [built-in envelopes](#built-in-envelopes) to easily to decode and unwrap events from sources like Kinesis, CloudWatch Logs, etc.
+
+### Validating with external references
+
+JSON schema [allows schemas to reference other schemas](https://json-schema.org/understanding-json-schema/structuring#dollarref) using the `$ref` keyword. The value of `$ref` is a URI reference, but you likely don't want to launch an HTTP request to resolve this URI. Instead, you can pass resolving functions through the `handlers` parameter:
```

Comment:
```suggestion
JSON Schema [allows schemas to reference other schemas](https://json-schema.org/understanding-json-schema/structuring#dollarref) using the `$ref` keyword with a URI value. By default, `fastjsonschema` will make a HTTP request to resolve this URI. 

You can use `handlers` parameter to have full control over how references schemas are fetched. This is useful when you might want to optimize caching, reducing HTTP calls, or fetching them from non-HTTP endpoints.

```

changes:

* **might not** over **likely don't**, as we can't assume their preferences given how diverse our customers are.
* _typo_: **JSON Schema** over **JSON schema**.
* _line editing_: merged sentences to give enough context on what happens by default first, followed by why they might want to use this.
* _addition_: added use cases so customers can make an informed decision whether this is for them or skip reading this section. Also helps searching later.